packages/components/Tabs/src/Tabs.tsx

```diff
@@ -82,10 +91,33 @@ export const Tabs = compose<TabsType>({
 
     const styleProps = useStyling(userProps, (override: string) => state[override] || userProps[override]);
 
+    const onKeyDown = (ev: IKeyboardEvent) => {
+      if (ev.nativeEvent.key === 'ArrowRight' || ev.nativeEvent.key === 'ArrowLeft') {
+        const length = state.context.enabledKeys.length;
+        const currTabItemIndex = state.context.enabledKeys.findIndex(x => x == state.context.selectedKey)
+        let newCurrTabItemIndex;
+        if (ev.nativeEvent.key === 'ArrowRight') {
+          if (!(!isCircularNavigation && currTabItemIndex + 1 == length)) {
+            newCurrTabItemIndex = (currTabItemIndex + 1) % length;
+            state.context.selectedKey = state.context.enabledKeys[newCurrTabItemIndex];
+            data.onKeySelect(state.context.selectedKey);
+          }
+        }
+        if (ev.nativeEvent.key === 'ArrowLeft') {
+          if (!(!isCircularNavigation && currTabItemIndex == 0)) {
+            newCurrTabItemIndex = (currTabItemIndex - 1 + length) % length;
+            state.context.selectedKey = state.context.enabledKeys[newCurrTabItemIndex];
+            data.onKeySelect(state.context.selectedKey);
+          }
+        }
+      }
+    };
+
     const slotProps = mergeSettings<TabsSlotProps>(styleProps, {
-      root: { rest, ref: componentRef, accessibilityLabel: accessibilityLabel, accessibilityRole: 'tablist'},
+      root: { rest, ref: componentRef, accessibilityLabel: accessibilityLabel, accessibilityRole: Platform.OS !== 'windows' ? 'tablist' : null, ...pressable.props}, // Add windows role when RN is at >= 0.64
       label: { children: label },
-      container: { isCircularNavigation: isCircularNavigation, defaultTabbableElement: selectedTabsItemRef },
+      container: Platform.OS !== 'windows' ? { isCircularNavigation: isCircularNavigation, defaultTabbableElement: selectedTabsItemRef } : null,
+      stack: Platform.OS !== 'windows' ? null : {focusable: true, ref: focusZoneRef, onKeyDown: onKeyDown}
     });
```

Comment:
why not make a Tabs.windows.ts? Or maybe a `windowsSlotProps` variable? It seems like there's a lot of platform checks that we could condense

Reply:
Since UWP and win32 have a lot similarities, I thought that having one file for both would be more convenient. I will add the windowsSlotProps variable.